hoprd/rest-api/src/node.rs

```diff
@@ -377,7 +383,9 @@ pub(super) async fn info(State(state): State<Arc<InternalState>>) -> Result<impl
                 channel_closure_period: channel_closure_notice_period.as_secs(),
                 indexer_block,
                 indexer_checksum,
-                index_block_prev_checksum,
+                // FIXME: this is only done for backwards-compatibility, ideally we don't return
+                // this value
+                index_block_prev_checksum: indexer_block,
```

Comment:
@coderabbitai Please add an issue to remove this functionality for milestone 3.0.